R-package/R/lgb.Booster.R

```diff
@@ -39,12 +39,12 @@ Booster <- R6::R6Class(
         # Check if training dataset is not null
         if (!is.null(train_set)) {
           # Check if training dataset is lgb.Dataset or not
-          if (!lgb.check.r6.class(train_set, "lgb.Dataset")) {
+          if (!lgb.check.r6.class(object = train_set,name = "lgb.Dataset")) {
```

Comment:
```suggestion
          if (!lgb.check.r6.class(object = train_set, name = "lgb.Dataset")) {
```

The linting jobs will complain about this missing space, sorry.

Reply:
Thank you for constantly checking the commits, and also improving my code style.